src/common/types/index.ts

```diff
@@ -4,14 +4,27 @@ export type APIResponse<T = void> = {
   message: string;
   errors: APIFieldError[];
   data: T;
+  pagination?: PaginationType;
 } | null;
 
+export type FilterType = {
+  page?: number;
+  limit?: number;
```

Comment:
ayoyyo, im looking for api params utilies not docs based, in fact it can be used like this

> {{baseURL}}/api/balance/history?limit=12&page=2&order_by=desc&sort=created_at

so this is the type

`export type FilterType = {
  order_by?: "desc" | "asc";
  sort?: string;
};`

`export type PaginationType = {
  page: number;
  limit: number;
  total_items: number;
  total_pages: number;
};`

`type BalanceHistoryParams = {
  limit?: PaginationType["limit"];
  page?: PaginationType["page"];
  order_by?: FilterType["order_by"];
  sort?: FilterType["sort"];
};`